contracts/core/RebalancingSetToken.sol

```diff
@@ -297,9 +296,9 @@ contract RebalancingSetToken is
 
         // Creating pointer to Core to Issue next set and Deposit into vault and to nextSet token
         // to transfer fees
-        ICore core = ICore(ISetFactory(factory).core());
+        ICore core = ICore(IRebalancingSetFactory(factory).core());
         ISetToken nextSetInstance = ISetToken(nextSet);
-        address protocolAddress = ICore(ISetFactory(factory).core()).protocolAddress();
+        address protocolAddress = ICore(IRebalancingSetFactory(factory).core()).protocolAddress();
```

Comment:
Looks like this line can just be `address protocolAddress = core.protocolAddress();` ?

Reply:
+1